schema/va-spec/base/va-spec-source.yaml

```diff
@@ -993,36 +991,30 @@ $defs:
         extends: predicate
         enum:
         - isOncogenicFor
-        - isProtectiveFor
-        - isPredisposingFor
+        - isProtectiveAgainst
       objectTumorType:
         extends: object
         oneOf:
         - $ref: "/ga4gh/schema/va-spec/1.x/base/json/Condition"
-        - $ref: "/ga4gh/schema/gks-core/1.x/json/MappableConcept"
+        - $ref: "/ga4gh/schema/gks-core/1.x/json/iriReference"
         description: >-
           The tumor type for which the variant impact is evaluated.
-      alleleOriginQualifier:
+      variantOriginQualifier:
         type: string
         description: >-
-          Reports whether the proposition should be interpreted in the context of an inherited
-          (germline) variant, an acquired (somatic) mutation, or both (combined).
-        enum:
-        - germline
-        - somatic
-        - combined
-      allelePrevalenceQualifier:
+          Reports whether the proposition should be interpreted in the context of a heritable
+          (germline) variant, an acquired (somatic) mutation, or more nuanced origin.
+        $ref: "/ga4gh/schema/gks-core/1.x/json/MappableConcept"
```

Comment:
@larrybabb note that some of our qualifiers had enums, where I am now updating them to MappableConcept. This is useful because even though some things seem obvious now (e.g. `germline`/`somatic`/`combined` for `allele origin`), there may be a desire in communities for more nuanced concepts (e.g. `somatic mosaicism` for `allele origin`) and not constraining to enums allows us to keep these fields broadly useful and reusable across current and future implementations and guidelines.